test/regexp/ferguson/readfre.chpl

```diff
@@ -19,8 +18,14 @@ writeln("#3 captures az, should match");
 ok = r.readf("%/([a-z]+)/", str);
 writeln(r.offset(), " ", ok, ":", str);
 writeln("#4 should not match");
-ok = r.readf("%/(0-9)/", str, error=error);
-writeln(r.offset(), " ", ok);
+
+try {
+  ok = r.readf("%/(0-9)/", str);
+} catch {
+  ok = false;
+  writeln(r.offset(), " ", ok);
```

Comment:
It's supposed to print out the offset either way; move it out of the catch block.